Source/Utilis/Protos/GenericMessage+Helper.swift

```diff
@@ -61,6 +61,52 @@ public extension GenericMessage {
 }
 
 extension GenericMessage {
+    public var messageContent: OneOf_Content? {
+        guard let content = content else {
+            return nil
+        }
+        return content
+    }
+    
+    public var messageData: MessageCapable? {
+        switch messageContent {
+        case .text(let data):
+            return data
+        case .confirmation(let data):
+            return data
+        case .reaction(let data):
+            return data
+        case .asset(let data):
+            return data
+        case .ephemeral(let data):
+            return data
```

Comment:
shouldn't we do something like this?

```suggestion
            return data.messageData
```